lib/language.js

```diff
@@ -110,1760 +110,1769 @@ function init() {
-      ,ro: 'Mediană'
+      ,sv: 'Median'
+      ,ro: 'Mediana'
       }
     ,'Readings' : {   
-      cs: 'Záznamů'
-      ,es: 'Valores'
+      cs: 'Záznamu'
       ,fr: 'Valeurs'
       ,pt: 'Valores'
+      ,sv: 'Avläsning'
       ,ro: 'Valori'
       }
     ,'StDev' : {   
       cs: 'St. odchylka'
-      ,es: 'Desviación estándar'
       ,fr: 'Déviation St.'
       ,pt: 'DesvPadr'
+      ,sv: 'StdDev'
       ,ro: 'Dev Std'
       }
     ,'Daily stats report' : {   
       cs: 'Denní statistiky'
-      ,es: 'Informe de estadísticas diarias'
       ,fr: 'Rapport quotidien'
       ,pt: 'Relatório diário'
-      ,ro: 'Raport statistică zilnică'
+      ,sv: 'Dygnsstatistik'
+      ,ro: 'Raport statistica zilnica'
       }
     ,'Glucose Percentile report' : {   
       cs: 'Tabulka percentil glykémií'
-      ,es: 'Informe de percetiles de glucemia'
       ,fr: 'Rapport precentiles Glycémie'
       ,pt: 'Relatório de Percentis de Glicemia'
+      ,sv: 'Glukosrapport i procent'
       ,ro: 'Raport percentile glicemii'
       }
     ,'Glucose distribution' : {   
       cs: 'Rozložení glykémií'
-      ,es: 'Distribución de glucemias'
       ,fr: 'Distribution glycémies'
       ,pt: 'Distribuição de glicemias'
-      ,ro: 'Distribuție glicemie'
+      ,sv: 'Glukosdistribution'
+      ,ro: 'Distribu?ie glicemie'
       }
     ,'days total' : {   
       cs: 'dní celkem'
-      ,es: 'Total de días'
       ,fr: 'jours totaux'
       ,pt: 'dias total'
+      ,sv: 'antal dagar'
       ,ro: 'total zile'
       }
     ,'Overall' : {   
       cs: 'Celkem'
-      ,es: 'General'
       ,fr: 'Dans l\'ensemble'
       ,pt: 'Geral'
+      ,sv: 'Genomsnitt'
       ,ro: 'General'
       }
     ,'Range' : {   
       cs: 'Rozsah'
-      ,es: 'Intervalo'
       ,fr: 'Intervalle'
       ,pt: 'intervalo'
+      ,sv: 'Intervall'
       ,ro: 'Interval'
       }
     ,'% of Readings' : {   
-      cs: '% záznamů'
-      ,es: '% de valores'
+      cs: '% záznamu'
       ,pt: '% de valores'
+      ,sv: '& av avläsningar'
       ,ro: '% de valori'
       }
     ,'# of Readings' : {   
-      cs: 'počet záznamů'
-      ,es: 'N° de valores'
+      cs: 'pocet záznamu'
       ,fr: 'nbr de valeurs'
       ,pt: 'N° de valores'
+      ,sv: 'av avläsningar'
       ,ro: 'nr. de valori'
       }
     ,'Mean' : {   
-      cs: 'Střední hodnota'
-      ,es: 'Media'
+      cs: 'Strední hodnota'
       ,fr: 'Moyenne'
       ,pt: 'Média'
+      ,sv: 'Genomsnitt'
       ,ro: 'Medie'
       }
     ,'Standard Deviation' : {   
       cs: 'Standardní odchylka'
-      ,es: 'Desviación estándar'
       ,fr: 'Déviation Standard'
-      ,pt: 'Desvio padrão'
-      ,ro: 'Deviație standard'
+      ,pt: 'Desvio padrão
+      ,sv: 'Standardavvikelse'
+      ,ro: 'Devia?ie standard'
       }
     ,'Max' : {   
       cs: 'Max'
-      ,es: 'Max'
       ,fr: 'Max'
       ,pt: 'Max'
+      ,sv: 'Max'
       ,ro: 'Max'
       }
     ,'Min' : {   
       cs: 'Min'
-      ,es: 'Min'
       ,fr: 'Min'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/860866771/issues/source?bid=2452379&fileBranchId=2818655#l623)